Source/Core/Core/Config/MouseSettings.h

```diff
@@ -0,0 +1,16 @@
+// Copyright 2018 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "Common/Config/Config.h"
+
+namespace Config
+{
+extern const Info<double> MOUSE_SENSITIVITY;
+static constexpr double scaling_factor_to_make_mouse_sensitivity_one_for_fullscreen = 2.0;
+extern const Info<double> SNAPPING_DISTANCE;
+extern const Info<unsigned short> CENTER_MOUSE_KEY;
+static constexpr unsigned short CENTER_MOUSE_KEY_UNBOUND_VALUE = 0xFFFF;
+extern const Info<bool> OCTAGONAL_MOUSE_JAIL_ENABLED;
```

Comment:
Comparing with the other *Settings files, I think these should all start with `MOUSE_`. So `MOUSE_SENSITIVITY` is fine, but the other three should probably be `MOUSE_SNAPPING_DISTANCE`, `MOUSE_CENTER_KEY`, `MOUSE_OCTAGONAL_JAIL_ENABLED`, or something like that.